### ISSUE TITLE
Unable to install global gulp package due vinyl-fs.

### ISSUE THREAD

**Issue by markbook2** [gulpjs/vinyl-fs#91]
I am trying to install gulp and can't do it due error that probably caused by vinyl-fs, here is an example:
```
$ npm install -g git+https://git@github.com/gulpjs/gulp.git
npm WARN engine gulp@3.9.0: wanted: {"node":">= 0.9"} (current: {"node":"5.0.0-pre","npm":"2.14.2"})
npm ERR! Linux 4.1.6-1-ARCH
npm ERR! argv "/opt/nodejs/bin/node" "/opt/nodejs/bin/npm" "install" "-g" "git+https://git@github.com/gulpjs/gulp.git"
npm ERR! node v5.0.0-pre
npm ERR! npm  v2.14.2
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: vinyl-fs@'>=0.3.0 <0.4.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.0.1","0.0.2","0.1.0","1.0.0"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'gulp'
npm ERR! notarget 

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/npm-debug.log
```
Another  attempt:
```
# /tmp $ npm install -g gulp                                      
npm WARN engine gulp@3.9.0: wanted: {"node":">= 0.9"} (current: {"node":"5.0.0-pre","npm":"2.14.2"})
npm ERR! Linux 4.1.6-1-ARCH
npm ERR! argv "/opt/nodejs/bin/node" "/opt/nodejs/bin/npm" "install" "-g" "gulp"
npm ERR! node v5.0.0-pre
npm ERR! npm  v2.14.2
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: vinyl-fs@'>=0.3.0 <0.4.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.0.1","0.0.2","0.1.0","1.0.0"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'gulp'
npm ERR! notarget 

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/npm-debug.log
```
I just compiled fresh node from the github repository and trying to install gulp:
```
git clone --depth 1 https://github.com/nodejs/node.git
cd node
./configure --prefix=/opt/nodejs && make && make install
/opt/nodejs/bin/npm install -g gulp
npm WARN engine gulp@3.9.0: wanted: {"node":">= 0.9"} (current: {"node":"5.0.0-pre","npm":"2.14.2"})
...
```
Other global packages like typescript, bower etc installed without any problem.

**Comment by phated** [comment]
Needs to be updated at the 0.3.13 tag.  See: https://github.com/wearefractal/vinyl-fs/blob/v0.3.13/package.json#L41

**Comment by markbook2** [comment]
Probably the right fix for this problem is update the gulp's package.json to depend from the more fresh version of the vinyl-fs without the engineStrict.

**Comment by contra** [comment]
@markbook2 the latest vinyl-fs has a ton of breaking changes that can't be brought back to 3.x of gulp, they are in the 4.0 branch though

**Comment by phated** [comment]
This has been tagged and published as 0.3.14.  You won't notice the commits in the master branch because I did them in a branch and just tagged/published from that to avoid a big rebase on this project.
